parser-typechecker/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -1,4 +1,5 @@
 {-# OPTIONS_GHC -fno-warn-partial-type-signatures #-}
+{-# OPTIONS_GHC -fwarn-incomplete-patterns #-}
```

Comment:
I compiled after adding the constructors for the new errors expecting to be guided to where to implement the messages and ... it compiled, then exploding in tests, so adding this here. Can always remove if not necessary.